documentation/src/main/asciidoc/configuration.adoc

```diff
@@ -203,7 +203,33 @@ Integer dbPort = ConfigResolver
 
 ==== Supported types
 
-The types supported out of the box include: String, Integer, Long, Float, Double, Boolean, Class.
+This table shows the types supported by Deltaspike already:
```

Comment:
On the type list I don't get all the choices (ex: why file and not path, why instant but not offsetdatetime which are more human friendly) but not a big blocker but the doc should document how to use it since as this DeltaSpike does *not* support the new type, it enables to use them if you do the work in your app - this is why I thought they should be matched in the extension and enabled implicitly ifuser didn't registered a custom producer (converter bean actually).

That's the big picture for me, minimum there: ensure the doc explains how to use this feature and flag in the table the manual converter as such vs the implicit ones (primitive wrappers, string, class).

rest is good for me

hope it makes sense.